src/components/switch/partial-styles/_readonly.scss

```diff
@@ -46,13 +46,20 @@
 
```

Comment:
There are two additional changes in the readme mode, that we might want to fix while we're at it:
- the font and size of the label has changed
- the "track" has become slightly longer, making the positioning of the checkmark icon a little bit off when the switch is `selected`

v32.4.4
![image](https://user-images.githubusercontent.com/154725/126611080-1c6d877b-e114-42d9-9c21-4bcccaa5d30d.png)

v33.0.0
![image](https://user-images.githubusercontent.com/154725/126611346-a8581548-3f7e-456a-b44c-7cc1c8d523a0.png)

Reply:
good catch. I'll fix